From ec6a223e9b667b4605fb8340db17b357749cdf12 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Thu, 15 May 2014 23:35:11 +0300
Subject: [PATCH 1/4] Document that @INC has /usr/lib/perl/5.18, not
 /usr/lib/perl/5.18.2

Since at least 5.8.4-8 (Debian sarge release), the Perl search
path for the core modules has used the major version (5.18)
instead of the full one (5.18.2).
---
 perl-policy.sgml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/perl-policy.sgml b/perl-policy.sgml
index 626c514..abe04d9 100644
--- a/perl-policy.sgml
+++ b/perl-policy.sgml
@@ -168,9 +168,11 @@
 	      <p>
 		Modules included in the core Perl distribution.
 	        <example>
-/usr/lib/perl/<var>version</var>
-/usr/share/perl/<var>version</var>
+/usr/lib/perl/<var>shortversion</var>
+/usr/share/perl/<var>shortversion</var>
 	        </example>
+		Where <var>shortversion</var> indicates the current Perl major
+		version (for example <tt>5.18</tt>).
 	      </p>
 	    </item>
 	    <tag><var>site</var> (old)</tag>
-- 
2.0.0.rc2

